### DISCUSSION TITLE
密码没错，一直这样

### DISCUSSION THREAD

**Discussion by yuehun** [AlistGo/alist#721] [Q&A]
### Alist Version / Alist 版本

2.1.3

### Describe the bug / 问题描述

密码没错，一直这样
![微信截图_20220310172712](https://user-images.githubusercontent.com/24507941/157631963-0ed9def8-9161-48a4-9462-e6443b543c37.png)

**Comment by xhofe** [comment]
这只能是密码错了。

**Comment by yuehun** [comment]
> 这只是密码错了。

我用./alist-xxxx -password查的密码，复制粘贴的，还是这样....

**Comment by xjc19960724** [comment]
注意空格

**Comment by xhofe** [comment]
- 检查是否反向代理缓存，直接源地址尝试
- 检查是否在正确的位置以正确的配置文件启动

**Reply by yuehun** [reply]
> * 判断是否反代理缓存，直接源地址尝试
> * 检查是否在正确的位置以正确的配置文件启动

手动./alist-XXX启动密码就对，守护进程systemctl start alist启动密码就不对是啥呀

**Reply by xhofe** [reply]
检查你的service文件，它读取了错误的位置的配置文件。

**Reply by yuehun** [reply]
> 检查您的服务文件，它读取了错误位置的配置文件。

![image](https://user-images.githubusercontent.com/24507941/157652994-1031ab32-730d-47fc-afb3-5b6d8219c164.png)
![image](https://user-images.githubusercontent.com/24507941/157653024-8aee0225-df85-41df-8a4f-529e5c218bfd.png)

**Reply by xhofe** [reply]
WorkingDirectory

**Reply by yuehun** [reply]
> 工作目录
![image](https://user-images.githubusercontent.com/24507941/157653467-903d2b56-58cf-4ef8-89b4-a45523e619f1.png)

**Reply by xhofe** [reply]
那你倒是改啊

**Reply by yuehun** [reply]
> 工作目录

默认的，应该不会错吧
![image](https://user-images.githubusercontent.com/24507941/157653573-629077f7-2235-47ab-8d7d-7edd94126b75.png)

**Reply by xhofe** [reply]
你的**service文件**，**WorkingDirectory**没有改

**Reply by yuehun** [reply]
> 你的**服务文件**，**WorkingDirectory**没有改

![image](https://user-images.githubusercontent.com/24507941/157654081-9129ff79-4501-4dc1-8367-e052f8a63dd2.png)
这样吗，我以为直接修改alist目录就可以了

**Reply by xhofe** [reply]
# 其中path_alist为alist所在的路径
说的够明白了吗

**Reply by yuehun** [reply]
> # 其中path_alist为alist所在的路径
> 说的够明白了吗

![image](https://user-images.githubusercontent.com/24507941/157654469-90983cef-3131-4092-9ee4-2823f71ee3a3.png)

**Reply by yuehun** [reply]
> # 其中path_alist为alist所在的路径
> 说的够明白了吗

没看到上面还有一个path_alist，我的，哈哈哈

**Comment by zsbai** [comment]
我前些天也遇到了这个问题，密码复制粘贴也不对 但是浏览器无痕模式可以进去 不开无痕就一直密码错误 感觉像是缓存的问题，今天又好了
